substrate/executor/src/native_executor.rs

```diff
@@ -46,6 +50,34 @@ fn gen_cache_key(code: &[u8]) -> u64 {
 	h.finish()
 }
 
+/// fetch a runtime version from the cache or if there is no cached version yet, create
+/// the runtime version entry for `code`, determines whether `RunWith::NativeRuntime`
+/// can be used by by comparing returned RuntimeVersion to `ref_version`
+fn fetch_cached_runtime_version<'a, E: Externalities>(
+	cache: &'a mut MutexGuard<CacheType>,
+	ext: &mut E,
+	code: &[u8],
+	ref_version: RuntimeVersion
+) -> &'a RunWith {
+	cache.entry(gen_cache_key(code))
+		.or_insert_with(||{
```

Comment:
nit: space after `||`

Reply:
we should really agree on a style guide (and have linter do that stuff for us) ;) .